CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0.0)
 project(cmake_wrapper)
 
-include(conanbuildinfo.cmake)
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
```

Comment:
Why this change?

Reply:
I use the package development flow when developing the recipes.
When building, this does not copy the sources from `self.source_folder` to `self.build_folder`,
so the include requires an explicit path.

Reply:
https://github.com/bincrafters/conan-templates/pull/49

Reply:
Merged into the templates